app/lib/video_metadata_extractor.rb

```diff
@@ -12,7 +12,7 @@ def initialize(path)
   rescue Terrapin::ExitStatusError, Oj::ParseError
     @invalid = true
   rescue Terrapin::CommandNotFoundError
-    raise Paperclip::Errors::CommandNotFoundError, 'Could not run the `ffprobe` command. Please install ffmpeg.'
+    raise Paperclip::Errors::CommandNotFoundError, I18n.t('ffmpeg.errors.command_not_found')
```

Comment:
This is not actually user-facing. If anything, it would appear in server logs, which should not be using the end user's locale.